packages/create-react-app-demo/src/docs-chat/ai.tsx

```diff
@@ -30,7 +21,8 @@ const docs = [
   },
 ];
 const corpus = new LocalCorpus(staticLoader(docs), defaultChunker);
-await corpus.startLoading();
```

Comment:
@juberti by `await`ing this at the top level, we blocked the entire UI from showing until it was complete. 

The better thing to do is to start the loading ASAP, but only block on it when we actually need to use the results.